bin/strap.sh

```diff
@@ -332,9 +346,20 @@ fi
 
 # Install from Daptiv Brewfile
 DAPTIV_BREWFILE="$HOME/.Daptiv.Brewfile"
+DAPTIV_BREWFILE_BLACKLIST="$DAPTIV_BREWFILE.blacklist"
 if [ -f "$DAPTIV_BREWFILE" ]; then
   log "Installing from Daptiv Brewfile:"
-  brew bundle check --file="$DAPTIV_BREWFILE" || brew bundle --file="$DAPTIV_BREWFILE"
+  if [ ! -f "$DAPTIV_BREWFILE_BLACKLIST" ]; then
+    brew bundle check --file="$DAPTIV_BREWFILE" || brew bundle --file="$DAPTIV_BREWFILE"
+  else
+    log "Using blacklist"
+    while read BLACKLIST_LINE
+    do
+      if [ ! -z $BLACKLIST_REGEX ]; then BLACKLIST_REGEX+="|"; fi
+      BLACKLIST_REGEX+="\"$BLACKLIST_LINE\""
+    done < $DAPTIV_BREWFILE_BLACKLIST
+    sed -E "/$BLACKLIST_REGEX/d" $DAPTIV_BREWFILE | brew bundle check --file=- || sed -E "/$BLACKLIST_REGEX/d" $DAPTIV_BREWFILE | brew bundle --file=-
```

Comment:
it might be easier to read if you did somethign like this: 

```
CLEAN_BREWFILE=$(sed -E "/$BLACKLIST_REGEX/d" $DAPTIV_BREWFILE)
echo "$CLEAN_BREWFILE" | brew bundle check --file=- || echo "$CLEAN_BREWFILE" | brew bundle --file=-
```

Reply:
actually, now that I think about it. run the blacklist replacement and save the result to `/tmp/daptiv.brewfile` or something, that way you can log the file if something goes wrong and it can be inspected.

Reply:
something like 
```
BREWFILE_CLEAN='/tmp/some-file-name'
log "Using clean brewfile: $BREWFILE_CLEAN"
sed -E "/$BLACKLIST_REGEX/d" $DAPTIV_BREWFILE > $BREWFILE_CLEAN
brew bundle check --file="$BREWFILE_CLEAN" || brew bundle --file="$BREWFILE_CLEAN"
```

Reply:
🔧 5476f9cd3b891e20225e45e902e1d4cd4b1bbd9a